core/bin/zksync_event_listener/src/monitor.rs

```diff
@@ -52,6 +52,8 @@ impl Handler<RemoveSubscriber> for ServerMonitor {
 impl Handler<NewEvents> for ServerMonitor {
     type Result = ();
 
+    //FIXME remove clippy rule.
```

Comment:
@franciscotobar Do you think we should create a task to address it?

Reply:
```suggestion
    // FIXME: remove clippy rule.
```

Reply:
We could, I dont think that is possible to apply the rule but a task could be created to prove that Im wrong :S

Reply:
Create task [Rollup-214](https://rsklabs.atlassian.net/browse/ROLLUP-214)